meta-resin-common/recipes-connectivity/resin-proxy-config/resin-proxy-config/redsocks.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=redsocks transparent proxy redirector
 Requires=mnt-boot.mount resin-proxy-config.service
```

Comment:
If it needs name resolution, shouldn't it be in requires too? After only defines order.